content/en/user-guide/ci/circle-ci/index.md

```diff
@@ -3,77 +3,437 @@ title: "CircleCI"
+          name: Export state
+          command: localstack state export ls-state.zip
+      - persist_to_workspace:
+        paths:
+          - ls-state.zip
+      # Store state as artifact for local debugging
+      - store_artifact:
+          key: ls-state
+          paths: ls-state.zip
+...
+  localstack-load-state:
+    executor: localstack/default
+    steps:
+      ...
+      # Localstack already running
+      - attach_workspace:
+          at: .
+      - run:
+          name: Import state
           command: |
-            pip3 install localstack awscli-local[ver1]
-            docker pull localstack/localstack
-            localstack start -d                     
+            test -f ls-state.zip && localstack state import ls-state.zip
+...
+  workflows:
+  localstack-build:
+    jobs:
+      - localstack-save-state
+      - localstack-load-state
+```
+More information about state import and export [here](/user-guide/state-management/export-import-state).
+
+#### Cache
+To preserve state between workflow runs, you can take leverage of CircleCI's caching too.
+This strategy will persist Localstack's state for every workflow re-runs, but not for different workflows.
+
+```yaml
+...
+jobs:
+  localstack-update-state:
+    executor: localstack/default
+    steps:
+      ...
+      # Localstack already running
+      # Let's restore previous workflow run's Localstack state
+      - restore_cache:
+          # Use latest "ls-state" prefixed cache
+          key: ls-state-
+      - run:
+          name: Import state
+          command: test -f ls-state.zip && localstack state import ls-state.zip
+      ...
+      # Infrastructure had been updated
+      # Let's update cached Localstack state
+      - run:
+          name: Export state
+          command: localstack state export ls-state.zip
+      - save_cache:
+          key: ls-state-{{checksum ls-state.zip}}
+          paths: ls-state.zip
+  ...
+  localstack-do-work:
+    executor: localstack/default
+    steps:
+      # Localstack already running
+      - restore_cache:
+          # Use latest "ls-state" prefixed cache
+          key: ls-state-
+      - run:
+          name: Import state
+          command: test -f ls-state.zip && localstack state import ls-state.zip
+      ...
+
+
+# Example workflows
+workflows:
+  localstack-build:
+    jobs:
+      - localstack-update-state
+      - localstack-do-work
+      ...
+```
+More information about state management [here](/user-guide/state-management/export-import-state).
+
+#### Cloud Pods
+Cloud pods providing an easy solution to persist Localstack's state, even between workflows or projects.
 
-            echo "Waiting for LocalStack startup..."  
-            localstack wait -t 30                     
-            echo "Startup complete"
-            
+Find more information about cloud pods [here](/user-guide/state-management/cloud-pods/).
+
+##### Multiple projects
+Update cloud pod in it's own project (ie IaC repo).
```

Comment:
> Update cloud pod in it's own project (ie IaC repo).

I am not 100% sure what "ie IaC repo" means. Maybe you could phrase this in a more verbose fashion?

Reply:
Here it means that the infrastructure code is stored in a separate repo which runs its own pipeline that creates/updates the cloud pod we want to use here.

Reply:
That makes sense, could you maybe explain this in the docs / make the sentence a bit more verbose?